packages/gatsby-www-components/src/components/guide-list.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+
+import docsHierarchy from "../data/sidebars/doc-links.yaml"
```

Comment:
These files are used by more components than just this (see `gatsby-node` and `item-list`). It doesn’t make sense to move these YAML files to a "components" file when they're more like data integral to the site itself. I would leave the `guide-list` component out of this initial PR until we figure out what to do.

Reply:
Ok, I removed them from PR